tools/spin-ec2/README.md

```diff
@@ -13,8 +13,14 @@ The following example shows how to deploy infrastructure agent canaries:
     export AWS_PROFILE=<profile>
     export export AWS_REGION=<region>
 
-    # Provision canaries
-    ./spin-ec2 canaries provision -v v<agent_version> -l <license_key>
+    # Provision canaries (ansible password is necessary for windows hosts)
+    ./spin-ec2 canaries provision -v v<agent_version> -l <license_key> -x <ansible_password> 
+    
+    # Provision canaries by platform ( linux | windows | macos )
+    # windows canaries
+    ./spin-ec2 canaries provision -v v<agent_version> -l <license_key> -p windows -x ansible_password 
+    # linux canaries
+    ./spin-ec2 canaries provision -v v<agent_version> -l <license_key> -p linux
```

Comment:
What about macos?

Reply:
those were just examples to show that in windows you need `ansible_password` and in linux you don't. All options are listed in line 19. I can add another example anyway :)